daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
```

Comment:
@huikang yes we can enhance libnetwork to satisfy such a need. But we have to make sure it fits the model properly so that the criu feature works seamlessly across all the plugins. The best way to to do is to bring in the criu related design into libnetwork core and have an API towards the driver that will support the use-case.